src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -63,6 +66,16 @@ public static int executeQuery(String query) throws InterruptedException, IOExce
     return 0;
   }
 
+  public static Optional<String> executeSelectQuery(String query) throws InterruptedException, IOException {
+    QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(query).build();
+    TableResult results = getBigQueryService().query(queryConfig);
+    String outputRowValue = StringUtils.EMPTY;
```

Comment:
This won't work with Optionals. We don't want an Optional that contains ""

Reply:
Changed.